Library/Homebrew/test/spec_helper.rb

```diff
@@ -202,16 +204,7 @@
   config.around do |example|
     Homebrew.raise_deprecation_exceptions = true
 
-    Formulary.clear_cache
-    Tap.each(&:clear_cache)
```

Comment:
@apainintheneck, this needs to stay.

Reply:
Is it really necessary to clear taps at the instance level if we're already clearing `Tap`, `CoreTap` and `CoreCaskTap` at the class level? I guess `.clear_cache` doesn't clear the core tap instances, right?

Reply:
That is the `@instance` instance variable.

Reply:
https://github.com/Homebrew/brew/pull/16760